### DISCUSSION TITLE
System.IO.FileNotFoundException  CefSharp.Core.Runtime.dll

### DISCUSSION THREAD

**Discussion by dengchenhua** [cefsharp/CefSharp#3705] [General]
I Haded install VC_redist.x64.exe and VC_redist.x86.exe  when i restart my computer,i get an error ,System.IO.FileNotFoundException  CefSharp.Core.Runtime.dll
I use vs2019 to develop

**Comment by amaitland** [comment]
Please confirm all the files listed at the link below are present. 

https://github.com/cefsharp/CefSharp/wiki/Output-files-description-table-%28Redistribution%29

If you require further assistance then please provide more detail. Version numbers, what machine you are seeing the problem on, etc.
